checker/tests/tainting/Issue2330.java

```diff
@@ -0,0 +1,15 @@
+import java.util.*;
+import org.checkerframework.checker.tainting.qual.*;
+
+public class Issue2330<T extends @Tainted Object> {
+    @SuppressWarnings("tainting")
```

Comment:
Could you add a comment why these suppressions are necessary? Would an annotation on the class declaration be an alternative?

Reply:
I added a comment.   If we add `@Untainted` to the class declaration, a `super.invocation.invalid` error will be issued.  Also, the problem in Issue 2330 was when the annotation for the class declaration and the constructor result are not the same.